### ISSUE TITLE
Action for :hover/:focus states

### ISSUE THREAD

**Issue by drudunn** [pa11y/pa11y#398]
Would be great to use pa11y to simulate hover/focus states, to test drop-down menus on non-touch devices. 

This could be an action option, similar to `click`, eg:
```
"actions": [
  "screen capture pa11y/homepage-1440.png",
  "hover element #nav li:nth-child(1)",
  "wait for element #nav li:nth-child(1) div to be visible",
  "screen capture pa11y/homepage-1440-dropdown.png"
]
```

I apologise if this isn't a new request, but I couldn't find any previous GitHub or StackOverflow issues/answers. The only mention of hover I could find [was about contrast here](https://github.com/pa11y/pa11y/blob/master/TROUBLESHOOTING.md#does-pa11y-test-the-contrast-of-link-hover-states)

**Comment by mgifford** [comment]
I agree this would be good. I've tried to get this into CSSLint but haven't done any coding around this https://github.com/CSSLint/csslint/issues/264

This is a common and annoying issue for keyboard only users. There are a lot of keyboard only users out there.

**Comment by dylanb** [comment]
@drudunn if you cannot trigger the menu to open with only the keyboard, then you have an accessibility problem that you must fix

**Comment by asos-jasonp** [comment]
I vote for this feature aswel
